src/swsssdk/configdb.py

```diff
@@ -93,10 +150,12 @@ def get_config(self):
         hashes = client.keys('*')
         data = {}
         for _hash in hashes:
-            table_name = _hash.split(':', 1)[0]
-            key = _hash.split(':', 1)[1]
-            if not data.has_key(table_name):
-                data[table_name] = {}
-            data[table_name][key] = client.hgetall(_hash)
+            tokens = _hash.split(':', 1)
+            if len(tokens) == 2:
```

Comment:
if number of tokens is not equal to 2, should we print out warning message? as we are not suppose to get such data from configdb?